<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片转为base64字符串</string>
  </property>
  <widget class="QPushButton" name="btn_select">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlb_pic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>451</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/HASEE/Pictures/壁纸/aa91f082ceca5fdd-180f1d94e5f2db20-b3208454148f56d5a2c58c843ab82c63.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="tb_base64">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>531</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始转换</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
